### ISSUE TITLE
[AL] Entity modification is registered as entity creation in the activity log

### ISSUE THREAD

**Issue by serena-huridocs** [huridocs/uwazidocs#6677]
When users make edits to entities, the activity log registers the indicated actions as 'Created entity'. Is there a way to change it to 'Entity edited' to avoid confusion?

**Comment by RafaPolit** [comment]
The call to the API for activity log is, apparently, returning this "semantic" property with:
```
"semantic": {
                "description": "Created entity",
                "action": "CREATE",
                "name": "Entity Title or name (klhca2oq78)",
                "extra": "of type Template"
            }
```

This is probably a backend issue.

If this is being stored in the DB (as opposed to being a post-processed and populated in the JSON request) this could be important.

**Comment by aphilop** [comment]
@serena-huridocs @RafaPolit @mfacar 
Is that still an issue?
Wasn't this fixed with https://github.com/huridocs/uwazi/pull/6881?
I can see now in Activity log entries with: UPDATE [user] Updated entity: [entity id] of type [template]

**Comment by mfacar** [comment]
Yes @aphilop, it was solved in #6881. As @RafaPolit pointed out, it is a post-processed output, however filtering by the method  'UPDATE' doesn't return results for entities, these can be returned now by including 'UPDATE' as the search term. In the last tech call, it was agreed to review this and other topics related to the activity log at the product call.

**Comment by aphilop** [comment]
Closing this as fixed.
